Serializer/DtoEncoder.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopping\ApiTKDtoMapperBundle\Serializer;
+
+use Exception;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\Serializer\Encoder\DecoderInterface;
+use Symfony\Component\Serializer\Encoder\EncoderInterface;
+use Throwable;
+
+/**
+ * Class DtoEncoder.
+ *
+ * Registered as serializer.encoder, this DtoEncoder allows serialization of exceptions
+ *
+ * @package Shopping\ApiTKDtoMapperBundle\Serializer
+ */
+class DtoEncoder implements EncoderInterface, DecoderInterface
+{
+    public const FORMAT = 'dto';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function encode($data, /* string */$format, array $context = [])
```

Comment:
As we only support PHP >=7.2, could you add return typehints here because it should always return a string?